packages/investment-pool/contracts/InvestmentPool.sol

```diff
@@ -830,6 +846,79 @@ contract InvestmentPool is IInitializableInvestmentPool, SuperAppBase, Context,
         totalInvestedAmount += _amount;
     }
 
+    /**
+     * @notice Get the amount of voting tokens to mint. It is determined by the period (seed/private/public)
+     */
+    function _getVotingTokensAmountToMint(uint256 _amount) internal view returns (uint256) {
+        uint256 returnedValue;
+
+        if (totalInvestedAmount < seedFundingLimit) {
+            // Seed funding
+            if (totalInvestedAmount + _amount <= seedFundingLimit) {
+                // Multiplier will be the same for all voting tokens
+                returnedValue = _amount * seedFundingMultiplier;
+            } else if (totalInvestedAmount + _amount <= softCap) {
+                // Multiplier is going to be different. That's why we need to calculate
+                // the amount which is going to be invested in seed funding and which in private funding
+
+                uint256 amountInSeedFunding = seedFundingLimit - totalInvestedAmount;
+                uint256 ticketsForSeedFunding = amountInSeedFunding * seedFundingMultiplier;
+
+                uint256 amountInPrivateFunding = _amount - amountInSeedFunding;
+                uint256 ticketsForPrivateFunding = amountInPrivateFunding *
+                    privateFundingMultiplier;
+
+                returnedValue = ticketsForSeedFunding + ticketsForPrivateFunding;
+            } else if (totalInvestedAmount + _amount <= hardCap) {
```

Comment:
I assume that this is the case where 1 investor surpasses both the seed fund amount and soft cap, and (potentially), maybe event funds an entire project by themselves, am I correct?

Reply:
Yes